src/main/java/com/dumbdogdiner/stickyapi/common/util/textures/Mobhead.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util.textures;
+
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Credit: All mob heads are borrowed from the vanillatweaks datapacks.
+ * @see <a href="https://vanillatweaks.net/">Vanilla Tweaks</a>
+ */
+@SuppressWarnings("unused")
+public enum Mobhead {
```

Comment:
This should be PascalCased (MobHead)

Reply:
For whatever reason I thought Mobhead was an item name/word, like Playerhead